configdialog/lxqtpageselectwidget.cpp

```diff
@@ -102,12 +103,14 @@ QSize PageSelectWidget::sizeHint() const
     int w = 0;
     for(int i=0; i< count(); ++i)
     {
-        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignLeft | Qt::TextWordWrap, item(i)->text());
+        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignHCenter | Qt::TextWordWrap, item(i)->text());
         w = qMax(w, rect.width());
     }
+    // consider the icon size too
+    w = qMax(w, QApplication::style()->pixelMetric(QStyle::PM_IconViewIconSize));
```

Comment:
But no matter if the `QApplication::style()` or `this->style()` is used, by not providing the widget to `pixelMetric()` call you are denying styling for this particular widget via StyleSheet. Why?